baselines/bigquery-storage/src/v1beta1/big_query_storage_client.ts.baseline

```diff
@@ -185,6 +185,7 @@ export class BigQueryStorageClient {
     this.bigQueryStorageStub = this._gaxGrpc.createStub(
         this._opts.fallback ?
           (this._protos as protobuf.Root).lookupService('google.cloud.bigquery.storage.v1beta1.BigQueryStorage') :
+          /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Do we need both `eslint-disable` and `tsline:disable`?

Reply:
Right! :) we just need eslint-disable, thank you! also removed tslint in base PR, thanks!

Reply:
it does not work without the latest version of gts. This PR was intended to make some pure template change before updating gts, but seems this is more and more overlapped with the gts PR, I may close this later on and resolve all related problems in the base PR: https://github.com/googleapis/gapic-generator-typescript/pull/343